### ISSUE TITLE
App Link: http://clinical.meteorapp.com gives 503 Service Unavailable No healthy endpoints to handle

### ISSUE THREAD

**Issue by schadha101** [clinical-meteor/software-development-kit#138]
App Link: http://clinical.meteorapp.com gives 503 Service Unavailable: No healthy endpoints to handle the request. If you're the owner of this app, see this article for more information: http://bit.ly/2uCFzOo

**Comment by awatson1978** [comment]
Yeah, that's turned off on purpose.  Finances have been tight since grad school, and that was a utility bill I decided to forgo.  I can bring it back online if people donate/pitch in.

**Comment by schadha101** [comment]
Hello Abigail,

 

What is bare minimum cost to run the server? 

We have two physical servers running,  we can host meteorapp on our servers. Let me know.

Running on shoe string we have little to spare.

 

My direct email is sanjay.chadha @ safetylabs.org.

 

Regards,

    Sanjay <https://www.linkedin.com/in/sanjaychadha101/> 

 


 <http://www.safetylabs.org/> SafetyLABS_LogoLow.png

www.safetylabs.org <http://www.safetylabs.org/>  - Safety in Innovation.

Direct:  +1 613 203-6162 (m), +1 613-800-9794 (d)

Email: sanjay.chadha@safetylabs.org

**Comment by jimilinuxguy** [comment]
I am willing to make a sustained monthly donation  as well.
